source/includes/api/_metadata.md

```diff
@@ -378,3 +378,9 @@ curl -X POST https://api.resourcewatch.org/v1/dataset/metadata/find-by-ids \
   ]
 }
 ```
+
+## Privileges
```

Comment:
Can you move this elsewhere? in all other API endpoints, permissions are described per endpoint, not in a separate section. I don't see a good reason to break that consistency